mono/mini/driver.c

```diff
@@ -2578,14 +2579,14 @@ mono_jit_set_aot_only (gboolean val)
 static void
 mono_runtime_set_execution_mode (MonoEEMode mode)
 {
-	memset (&mono_ee_features, 0, sizeof (mono_ee_features));
-
 	switch (mode) {
 	case MONO_AOT_MODE_LLVMONLY:
 		mono_aot_only = TRUE;
 		mono_llvm_only = TRUE;
 
+#ifndef TARGET_WASM
 		mono_ee_features.use_aot_trampolines = TRUE;
+#endif
```

Comment:
what do you think about that @kumpera ? would you prefer to have another AOT mode so we can get rid of the `ifdef`?